vendor/github.com/docker/distribution/vendor.conf

```diff
@@ -1,43 +0,0 @@
-github.com/Azure/azure-sdk-for-go 088007b3b08cc02b27f2eadfdcd870958460ce7e
```

Comment:
Wait, why did all the nested vendor.conf files disappear?

@ndeloof could it be that you used an older version of `vndr`?


I just ran `vndr` on the PR and got this change;


[revendor.patch.txt](https://github.com/docker/cli/files/1536213/revendor.patch.txt)